api/wallet/views.py

```diff
@@ -1,3 +1,89 @@
 from django.shortcuts import render
+from rest_framework import generics, permissions
+from rest_framework.views import Response, status
+from .serializers import WalletSerializer
+from .models import Wallet
+from django.contrib.auth.models import User
+from rest_framework_jwt.settings import api_settings
+from .decorators import validate_request_data
+
+
+
+# Get the JWT settings
+jwt_payload_handler = api_settings.JWT_PAYLOAD_HANDLER
+jwt_encode_handler = api_settings.JWT_ENCODE_HANDLER
 
 # Create your views here.
+
+
+class WalletListView(generics.ListCreateAPIView):
+    """
+    GET wallets/
+    POST wallet/
```

Comment:
add an **s** here since we are using generics where the url is generated for us, since both of these are under the same view they will all have the same url but different REST methods (POST,GET)